bot_execute.py

```diff
@@ -245,15 +245,8 @@ async def inner():
     sql = "UPDATE fetched_blocks SET block_number = {0} WHERE network_name = '{1}' AND dex = '{2}' AND bot = '{3}' AND contract_instance = '{4}';".format(BLOCK_NUMBER, NETWORK_NAME, DEX, BOT, dca_bot_address)
     batch_sql.append(sql)
 
-    try:
-        CON.execute("BEGIN;")
-        for query in batch_sql:
-            CON.execute(query)
-    except:
-        CON.rollback()
-        raise
-    finally:
-        CON.commit()
+    for query in batch_sql:
+        CON.execute(query)
 
```

Comment:
We need to add `CON.commit()` here because CON.execute() doesn't update DB with itself only. I already tested it.